### PR TITLE
[script.module.pymysql] 0.8.0

### PR DESCRIPTION
**Pull request by smitchell6879**
### Add-on details:
- **General**
  - Add-on name:  Special Features
  - Add-on ID: plugin.video.specialfeatures
  - Version number: 3.0.0
  - Kodi/repository version: 17 & 18

- **Code location**
  - URL: https://github.com/smitchell6879/plugin.video.specialfeatures/tree/master
  - Release URL : https://github.com/smitchell6879/plugin.video.specialfeatures/releases/tag/3.0.0
  - Wiki URL :https://kodi.wiki/view/Add-on:Special_Features
  - Forum URL: https://forum.kodi.tv/showthread.php?tid=327042
  - Revision/tag: 3.0.0
  - Branch: master

### Description of changes:
Given a properly structured 'Extras' directory, this addon will present all of the bonus media in your library. 

Note: I attempted two pull request one creating my own branch and another directly to krypton but bother fail due to invalid permissions and since I am new to the whole process figured this would be the next attempt.

### Checklist:
- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### Note:
This is my first time I hope I am doing it right

### CODE REVIEW

**Comment by Lunatixz** [comment]
@smitchell6879 thanks for the PR, did you check to make sure your specialfeature plugin works with this module?

File: script.module.pymysql/addon.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.pymysql"
+       name="pymysql"
+       version="0.8.0"
+       provider-name="smitchell6879">
```
Comment (enen92) [inline script.module.pymysql/addon.xml]: Author should be the original module author not the one packaging it for kodi

File: script.module.pymysql/addon.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  <requires>
+    <import addon="xbmc.python" version="2.24.0"/>
```
Comment (enen92) [inline script.module.pymysql/addon.xml]: Krypton repository requires `xbmc.python` 2.25.0

File: script.module.pymysql/addon.xml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  </requires>
+  <extension point="xbmc.python.module"
+             library="lib" />
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en">Simple MySql Connector for Python</summary>
```
Comment (enen92) [inline script.module.pymysql/addon.xml]: Please use `en_GB`

File: script.module.pymysql/addon.xml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    <description lang="en">Simple MySql Connector for Python</description>
+    <platform>all</platform>
+    <language></language>
```
Comment (enen92) [inline script.module.pymysql/addon.xml]: language is not neeeded (only for content addons)

File: script.module.pymysql/addon.xml (hunk 1 above)
Comment (smitchell6879) [reply script.module.pymysql/addon.xml]: Where do I change this at? Also the Project has 72 contributors so would the project name 'PyMySQL' work?

File: script.module.pymysql/addon.xml (hunk 2 above)
Comment (smitchell6879) [reply script.module.pymysql/addon.xml]: This can be bumped up I just double checked it will work with 2.25.

**Comment by smitchell6879** [comment]
@enen92  I am new to all of this and just wanted to know did you fix the issues you found are did they get fixed when I pushed a update to my forked version some how?

**Comment by enen92** [comment]
@smitchell6879 when you have a pull request opened from one of your branches, any commit you do to the branch will reflect on the PR. I'll check don't worry

File: script.module.pymysql/addon.xml (hunk 1 above)
Comment (enen92) [reply script.module.pymysql/addon.xml]: https://pypi.python.org/pypi/PyMySQL

 Author: INADA Naoki

**Comment by smitchell6879** [comment]
@enen92  Thanks I am learning I had no idea that part would be that simple.

**Comment by smitchell6879** [comment]
Also will this or can this get added to the Leia branch as well? Or do I need to start a new pull request for that?

**Comment by Rechi** [comment]
It will get added to all higher versions, look for `Which repos are used` at https://kodi.wiki/view/Submitting_Add-ons

**Comment by smitchell6879** [comment]
Thanks I had read that but I wasn't sure if that was still true or not but again thanks.

**Comment by enen92** [comment]
@smitchell6879 when you submit to a repository let's say Krypton it will be available for any kodi version above krypton - so leia users will be able to use the plugin as well. Note that in the Leia repository only py2 and py3 compatible addons will be accepted (although this not affects you since your addon is in krypton branch). 
From Kodi19 only python3 addons will be available so I guess at that point any of the existing addons need to be ported and submitted to the 19 branch. In the meanwhile there is a migration repository. We are still far from that point though...but we as devs should keep this in mind for the future of Kodi.
Cheers

**Comment by smitchell6879** [comment]
@enen92 I had momentarily forgot about the py3... at one time in the earlier stages of this add-on I did test it and it was working but that was early on.  I will keep that in mind however as the py3 branch becomes more stable.. since this is my first real script of any kind I shied away from the py3 builds as when I was testing it was really unstable, the logs wouldn't report correctly and it would just crash at random times without leaving any callback or error in the log even attempt to figure it out.

But again thank you for taking time to review this and hopefully this will just be a start; as I have more ideas for the future.

**Comment by Lunatixz** [comment]
LGTM

**Pull request merged**